mesonbuild/build.py

```diff
@@ -278,6 +278,13 @@ def check_unity_compatible(self):
                                      'in Unity builds. You can only extract all '
                                      'the object files for each compiler at once.')
 
+    def get_outputs(self, backend):
+        # TODO: Consider if we need to handle genlist here
+        return [
+            backend.object_filename_from_source(self.target, source)
+                for source in self.srclist
```

Comment:
__[Flake8]__ [E131] continuation line unaligned for hanging indent ([link](https://sider.review/gh/repos/19784232/pulls/5695#issue-4359075))

<sub>posted by Sider</sub>

Reply:
Fixed in 0caa454b